### PR TITLE
OS-40: Added Claim

### PR DESCRIPTION
**Pull request by bartekkaminski**
I added a new module to the project

TASK:
https://openimis.atlassian.net/browse/OS-40

### CODE REVIEW

File: OpenImis.ModulesV2/ClaimModule/Logic/ClaimLogic.cs [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+        private IConfiguration _configuration;
+        protected IClaimRepository claimRepository;
+
+        public ClaimLogic(IConfiguration configuration)
         {
+            _configuration = configuration;
+            claimRepository = new ClaimRepository(_configuration);
+        }
+
+        public int RegisterClaim(Claim claim)
```
Comment (dragos-dobre) [inline OpenImis.ModulesV2/ClaimModule/Logic/ClaimLogic.cs]: In FamilyLogic you are naming the method Create. To be consistent, can we use the same method names between controllers, logics and repositories? I suggest using the CRUD naming: create, read, update and delete to start with.

Comment (bartekkaminski) [reply OpenImis.ModulesV2/ClaimModule/Logic/ClaimLogic.cs]: @dragos-dobre  It's a good idea. I changed the name

File: OpenImis.ModulesV2/ClaimModule/Repositories/ClaimRepository.cs [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            _configuration = configuration;
+            _hostingEnvironment = hostingEnvironment;
+        }
+
+        public int Create(Claim claim)
+        {
+            try
+            {
+                var XML = claim.XMLSerialize();
+                var RV = 2;
```
[the remark this answers is not in the document]
Comment (bartekkaminski) [reply OpenImis.ModulesV2/ClaimModule/Repositories/ClaimRepository.cs]: Done

File: OpenImis.ModulesV2/ClaimModule/Models/RegisterClaim/Claim.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+namespace OpenImis.ModulesV2.ClaimModule.Models.RegisterClaim
+{
+    public class Claim
+    {
+        public Details Details { get; set; }
+        public List<Item> Items { get; set; }
+        public List<Service> Services { get; set; }
+    }
+
+    public class Details
```
[the remark this answers is not in the document]
Comment (bartekkaminski) [reply OpenImis.ModulesV2/ClaimModule/Models/RegisterClaim/Claim.cs]: Done

**Pull request merged**